packages/secp256k1/secp256k1.0.4.4/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+synopsis: "Elliptic curve library secp256k1 wrapper for Ocaml"
+description: """\
+This library wrap the secp256k1 EC(DSA) library into an OCaml library. At the moment only a subset of functionalities are available:
+- Context: create, clone, destroy, randomize
+- Elliptic curve: public key creation
+- ECDSA: verify, sign, recover
+
+All exchanged data (pubkey, signature, seckey) are represented as hex strings."""
+maintainer: "Davide Gessa <gessadavide@gmail.com>"
+authors: [
+  "Davide Gessa <gessadavide@gmail.com>"
+  "Vincent Bernardoff <vb@luminar.eu.org>"
+  "Yoichi Hirai <i@yoichihirai.com>"
+  "Anton Trunov"
+]
+license: "MIT"
+homepage: "https://github.com/dakk/secp256k1-ml"
+bug-reports: "https://github.com/dakk/secp256k1-ml/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "1.3"}
+  "base" {build & >= "v0.9.2"}
```

Comment:
I think requiring `base >= "v1.12.0"` should also fix the issue, without requiring a more restrictive bound on dune

Reply:
In any case I sent your way a PR to simplify the build dependencies

Reply:
Sorry, there was a typo, I meant "v0.12.0". In any case, that drops ocaml 4.02, I think your original suggestion was better

Reply:
I think https://github.com/dakk/secp256k1-ml/pull/21/ is a strictly better approach from this point of view